lib/subspace/commands/provision.rb

```diff
@@ -1,5 +1,5 @@
 class Subspace::Commands::Provision < Subspace::Commands::Base
-  PASS_THROUGH_PARAMS = ["tags", "start-at-task"]
+  PASS_THROUGH_PARAMS = ["tags", "start-at-task", "private-key"]
```

Comment:
What does this do?

Reply:
Make it possible to specify private key to use when running playbook